.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    # Workflow files stored in the default location of `.github/workflows`. (You don't need to specify `/.github/workflows` for `directory`. You can use `directory: "/"`.)
+    directory: "/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+    allow:
+      # Allow both direct and indirect updates for all packages
+      - dependency-type: "all"
+
+  # Maintain dependencies for pip
+  # poetry is not a valid entry for package-ecosystem
+  - package-ecosystem: "pip"
```

Comment:
Sadly it looks like dependabot doesn't update pyproject yet, see https://github.com/dependabot/dependabot-core/issues/8603 :/ 
Anyway, we can give this a try.

Reply:
Yeah, it's too bad that it doesn't. Seems like the issue is is getting close to being solved so we can stay tuned on that.